lettermgmt/migrations/8.0.2.0/post-migration.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2016 Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    if version is None:
+        return
+
+    # Migrate states ['created', 'validated'] back to 'draft'
+    cr.execute("""UPDATE res_letter SET state = 'draft'
+                  WHERE state IN ('created', 'validated')""")
+
+    # Migrate 'letter.class' to 'letter.category'
+    cr.execute("""INSERT INTO letter_category SELECT * FROM letter_class""")
```

Comment:
I couldn't do it in pre-migrate. I tried a few approaches and it kept giving me errors.